cmd/crc-embedder/cmd/embed.go

```diff
@@ -53,7 +54,7 @@ func runEmbed(args []string) {
 		logging.Fatalf("Failed to download data files: %v", err)
 	}
 
-	bundlePath := path.Join(bundleDir, constants.GetDefaultBundleForOs(goos))
+	bundlePath := path.Join(bundleDir, constants.GetDefaultBundleForOs(goos, version.GetBundleVersion()))
```

Comment:
When I see 'default bundle', to me it means 'version of the bundle which we hardcoded at build time'. If you can override the version in a call to `GetDefaultBundleForOs`, it's no longer a default and the name has to be changed,  it's just `GetBundleNameForOs`